src/operator/controllers/vizier_controller.go

```diff
@@ -126,6 +127,34 @@ func getLatestVizierVersion(ctx context.Context, client cloudpb.ArtifactTrackerC
 	return resp.Artifact[0].VersionStr, nil
 }
 
+// validCSIDriver checks if the user is running an EKS cluster, and if so, whether they have
+// the CSIDriver enabled such that persistent volumes can be deployed.
+func validCSIDriver(clientset *kubernetes.Clientset, k8sVersion string) (bool, error) {
```

Comment:
I think returning an error from this func doesn't make much sense since we log it in this func and from the caller too, we should do one or the other.

Reply:
Also consider renaming this func, logically it's "(doesn't need CSI) or (needs CSI and has CSI)"
perhaps `validCSIDriverIfRequired` or something?
or perhaps a negation is better. i.e. a func that detects `missingNecessaryCSIDriver`